src/app/projects/projlist-map/projlist-map.component.ts

```diff
@@ -383,6 +377,26 @@ export class ProjlistMapComponent implements AfterViewInit, OnChanges, OnDestroy
     });
   }
 
+  /**
+   * Checks that the marker coordinates are valid and within bounds,
+   * then sets the pop-up message content, creates the marker icon,
+   * adds a click handler for the pop-up, and adds the new layer to the
+   * list of map layers.
+   * 
+   * @param proj The project to add the marker for.
+   */
+  private addMarkerToMap(proj: Project): void {
+    if (this.utils.markerMeetsConditions(proj)) {
+      const title = `${proj.name}\n` + `${proj.overlappingRegionalDistricts}\n`;
```

Comment:
Curious about the usage of \n here. Will this work under all conditions?

Reply:
Should be! Template literals support newline characters: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings

This old logic was moved into its own function and I believe it's been working fine up until now.